.travis.yml

```diff
@@ -1,5 +1,9 @@
 language: node_js
 node_js:
   - 'stable'
-  - '0.12'
-  - '0.10'
+  - '6.0.0'
```

Comment:
how did the old test pass, if the node version was `0.12` and `0.10`? there doesn't seem to be a compile step in the test command.